src/modules/75-cf/pages/feature-flags-detail/flag-pipeline-tab/components/available-pipelines-drawer/AvailablePipelinesDrawer.module.scss

```
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+.drawer {
+  background-color: var(--primary-bg);
+  max-height: 100vh;
+  overflow-y: scroll;
+}
+
+.footer {
+  position: fixed;
+  bottom: 0%;
```

Super-minor, but you don't need a unit type for `0`
```suggestion
  bottom: 0;
```

---

changed to `-0%` :P j/k fixed